### PR TITLE
T1000 e initialization

### PR DESCRIPTION
**Pull request by Dylanliacc**
Update V0.9.0

### CODE REVIEW

File: variants/tracker-t1000-e/platformio.ini [hunk 1]
```diff
@@ -0,0 +1,15 @@
+; tracker-t1000-e  v0.9.0
+[env:tracker-t1000-e]
+extends = nrf52840_base
+board = tracker-t1000-e
+board_level = extra
+; platform = https://github.com/maxgerhardt/platform-nordicnrf52#cac6fcf943a41accd2aeb4f3659ae297a73f422e
+build_flags = ${nrf52840_base.build_flags} -Ivariants/tracker-t1000-e -DTRACKER_T1000_E  -DRADIOLIB_GODMODE
```
Comment (fifieldt) [inline variants/tracker-t1000-e/platformio.ini]: 覺得人會抗議GODMODE. 有可能需要在RadioLib Upstream改改

File: variants/tracker-t1000-e/run_once.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```
Comment (fifieldt) [inline variants/tracker-t1000-e/run_once.sh]: 這檔案沒錯誤，但可能不需要加。在meshtastic使用bin/build-nrf52.sh (已經有mergehex和softdevice).

File: variants/tracker-t1000-e/variant.h [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+#define ADC_RESOLUTION 14
+#define BATTERY_SENSE_RESOLUTION_BITS 12
+// #define BATTERY_SENSE_RESOLUTION 4096.0
+
+#undef AREF_VOLTAGE
+#define AREF_VOLTAGE 3.0
+#define VBAT_AR_INTERNAL AR_INTERNAL_3_0
+
+// #define ADC_CTRL (32 + 6)
```
Comment (fifieldt) [inline variants/tracker-t1000-e/variant.h:138]: consider removing comment before submitting patch

File: variants/tracker-t1000-e/variant.h [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+// #define GPS_THREAD_INTERVAL 50
+#define GPS_FIX_HOLD_TIME 15000 // ms
+
+#define BATTERY_PIN 2
+// #define ADC_CHANNEL ADC1_GPIO2_CHANNEL
+#define ADC_MULTIPLIER (2.0F)
+
+#define ADC_RESOLUTION 14
+#define BATTERY_SENSE_RESOLUTION_BITS 12
+// #define BATTERY_SENSE_RESOLUTION 4096.0
```
Comment (fifieldt) [inline variants/tracker-t1000-e/variant.h:132]: consider removing comment before submitting patch

File: variants/tracker-t1000-e/variant.h [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+#define GPS_VRTC_EN (0 + 8) // P0.8, awlays high
+#define GPS_SLEEP_INT (32 + 12) // P1.12, awlays high
+#define GPS_RTC_INT (0 + 15) // P0.15, normal is LOW, wake by HIGH
+#define GPS_RESETB_OUT (32 + 14) // P1.14, awlays input pull_up
+
+// #define GPS_THREAD_INTERVAL 50
+#define GPS_FIX_HOLD_TIME 15000 // ms
+
+#define BATTERY_PIN 2
+// #define ADC_CHANNEL ADC1_GPIO2_CHANNEL
```
Comment (fifieldt) [inline variants/tracker-t1000-e/variant.h:127]: consider removing comment before submitting patch

File: variants/tracker-t1000-e/variant.h [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _VARIANT_WIO_SDK_WM1110_
```
Comment (fifieldt) [inline variants/tracker-t1000-e/variant.h:19]: is a new VARIANT name needed?

File: boards/tracker-t1000-e.json [hunk 7]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (fifieldt) [inline boards/tracker-t1000-e.json:1]: Looks good.

File: variants/tracker-t1000-e/platformio.ini (hunk 1 above)
Comment (fifieldt) [inline variants/tracker-t1000-e/platformio.ini]: Does this need  -Isrc/platform/nrf52/softdevice -Isrc/platform/nrf52/softdevice/nrf52 ?

File: variants/tracker-t1000-e/variant.cpp [hunk 8]
```diff
@@ -0,0 +1,64 @@
+/*
```
Comment (fifieldt) [inline variants/tracker-t1000-e/variant.cpp:1]: Looks good

File: src/modules/Telemetry/Sensor/T1000xSensor.cpp [hunk 9]
```diff
@@ -0,0 +1,128 @@
+#include "configuration.h"
```
Comment (fifieldt) [inline src/modules/Telemetry/Sensor/T1000xSensor.cpp:1]: ok!

File: src/modules/Telemetry/Sensor/T1000xSensor.h [hunk 10]
```diff
@@ -0,0 +1,24 @@
+#include "configuration.h"
```
Comment (fifieldt) [inline src/modules/Telemetry/Sensor/T1000xSensor.h:1]: ok!

File: src/modules/TextMessageModule.cpp [hunk 11]
```diff
@@ -12,7 +12,11 @@ ProcessMessage TextMessageModule::handleReceived(const meshtastic_MeshPacket &mp
     auto &p = mp.decoded;
     LOG_INFO("Received text msg from=0x%0x, id=0x%x, msg=%.*s\n", mp.from, mp.id, p.payload.size, p.payload.bytes);
 #endif
-
+#ifdef PIN_BUZZER
```
Comment (fifieldt) [inline src/modules/TextMessageModule.cpp:15]: This turns on the buzzer for all variants. Is this what the community wants?

File: src/gps/GPS.cpp [hunk 12]
```diff
@@ -400,6 +400,12 @@ bool GPS::setup()
     int msglen = 0;
 
     if (!didSerialInit) {
+#ifdef GNSS_Airoha
```
Comment (fifieldt) [inline src/gps/GPS.cpp:403]: See https://github.com/fifieldt/meshtastic-firmware/commit/7b6b960671e1e2787003bd903019e20f8be1cf5b

File: variants/tracker-t1000-e/platformio.ini (hunk 1 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/platformio.ini]: For now, let's do it this way. If anyone has any objections, we can make changes later~

File: variants/tracker-t1000-e/run_once.sh (hunk 2 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/run_once.sh]: done deleted

File: variants/tracker-t1000-e/variant.h (hunk 3 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/variant.h:138]: removed that

File: variants/tracker-t1000-e/variant.h (hunk 4 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/variant.h:132]: removed that

File: variants/tracker-t1000-e/variant.h (hunk 5 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/variant.h:127]: done for that

File: variants/tracker-t1000-e/variant.h (hunk 6 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/variant.h:19]: changed it to _VARIANT_TRACKER_T1000_E_

File: variants/tracker-t1000-e/platformio.ini (hunk 1 above)
Comment (Dylanliacc) [reply variants/tracker-t1000-e/platformio.ini]: changed it to the softdevice now

**Pull request closed without merging**